qiskit/opflow/primitive_ops/pauli_op.py

```diff
@@ -220,7 +220,7 @@ def eval(self,
                 corrected_z_bits = self.primitive.z[::-1]  # type: ignore
 
                 for bstr, v in front.primitive.items():
-                    bitstr = np.asarray(list(bstr)).astype(np.int).astype(np.bool)
+                    bitstr = np.asarray(list(bstr)).astype(int).astype(bool)
```

Comment:
I proposed the following in another PR.
```suggestion
                    bitstr = np.fromiter(bstr, dtype=int).astype(bool)
```

Reply:
Yes, thank you in another PR. That fixes PauliSumOp, but I forgot it for PauliOp. I know this is correct. So I commit this.